src/luos.c

```diff
@@ -44,10 +44,8 @@ static int luos_msg_handler(module_t* module, msg_t* input, msg_t* output) {
     if ((input->header.cmd == LUOS_REVISION) & (input->header.size == 0)) {
         output->header.cmd = LUOS_REVISION;
         output->header.target_mode = ID;
-#ifndef LUOS_REV
-#define LUOS_REV "V0.6"
-#endif
```

Comment:
You should keep the unknown thing because if someone fail to pass it we can get it more easily.